tests/types.ts

```diff
@@ -0,0 +1,42 @@
+export interface IApiResponse<T> {
+    data: T;
+}
+
+export interface IUser {
+    id: string;
+    username: string;
+    roleId: string;
+    firstName?: string;
+    lastName?: string;
+    phoneNumber?: string;
+    email?: string;
+    telegram?: string;
+}
+
+export interface IUserRequest {
```

Comment:
```suggestion
export interface IUserRequest = Omit<IUser, "id">
```
[Doc](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys)

Reply:
Otherwise we need to update related interface once User is updated.